# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccessibilityService.h
# Commit: 6591f14bbc71
# Full Hash: 6591f14bbc713f11b6316712d931ecc19a443a5c
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-05 03:52:32
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 06dda9072cb7 -r 6591f14bbc71 accessible/base/nsAccessibilityService.h
--- a/accessible/base/nsAccessibilityService.h	Mon Oct 04 19:28:49 2021 +0000
+++ b/accessible/base/nsAccessibilityService.h	Mon Oct 04 19:46:54 2021 +0000
@@ -230,6 +230,12 @@
   void NotifyOfImageSizeAvailable(mozilla::PresShell* aPresShell,
                                   nsIContent* aContent);
 
+  void NotifyOfPossibleBoundsChange(mozilla::PresShell* aPresShell,
+                                    nsIContent* aContent);
+
+  void NotifyOfResolutionChange(mozilla::PresShell* aPresShell,
+                                float aResolution);
+
   // nsAccessibiltiyService
 
   /**